examples/vector_tile_dragndrop.html

```diff
@@ -0,0 +1,82 @@
+<html>
+    <head>
+        <title>Itowns - Globe + vector-tiles</title>
+
+        <meta charset="UTF-8">
+        <link rel="stylesheet" type="text/css" href="css/example.css">
+        <link rel="stylesheet" type="text/css" href="css/LoadingScreen.css">
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.6/dat.gui.min.js"></script>
+    </head>
+    <body>
+        <div id="viewerDiv"></div>
+        <div id="description">
+            Drag and drop a valid style file for a vector tiles stream. The
+            style <u><b>must</b></u> contain the sources for the stream (<a target="_blank"
+                href="https://docs.mapbox.com/mapbox-gl-js/style-spec/sources/">see
+                here</a>). You can add other styles without having to reload the
+            page, as the previous layers will be hidden.  However, note that
+            having multiple styles may severly impact performances.</div>
+        <script src="js/GUI/GuiTools.js"></script>
+        <script src="../dist/itowns.js"></script>
+        <script src="js/GUI/LoadingScreen.js"></script>
+        <script>
+            var placement = {
+                coord: new itowns.Coordinates('EPSG:4326', 2.475, 48.807),
+                range: 120000,
+            }
+
+            var viewerDiv = document.getElementById('viewerDiv');
+            var view = new itowns.GlobeView(viewerDiv, placement);
+            var menu = new GuiTools('menuDiv', view, 300);
+
+            setupLoadingScreen(viewerDiv, view);
+
+            var currentLayer = {};
+
+            // Drag and drop handler
+            var fileReader = new FileReader();
+
+            var id = 0;
+            fileReader.onload = function onload(e) {
```

Comment:
it's possible to use `itowns.fetch.json()`?

Reply:
Not when doing this, because the file doesn't have an URL to fetch from.